pennylane/pauli/conversion.py

```diff
@@ -357,7 +367,7 @@ def is_pauli_sentence(op):
     """Returns True of the operator is a PauliSentence and False otherwise."""
     if op.pauli_rep is not None:
         return True
-    if isinstance(op, Hamiltonian):
+    if isinstance(op, (Hamiltonian, LinearCombination)):
```

Comment:
```suggestion
    if isinstance(op, Hamiltonian):
```

Should instead be covered by the `op.pauli_rep` check on the previous line.